starter-project-web-vue2/pages/andualem/index.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <v-app>
+    <v-dialog max-width="600px">
+        <template v-slot:activator="{ on }">
+            <v-btn v-on="on">
+                Add Blog
+            </v-btn>
+        </template>
+
+        <v-card>
+            <v-card-title>
+                Add a New Blog
+            </v-card-title>
+            <v-card-text>
+                 <v-text-field
+                v-model="blogTitle"
+                :value="text-title"
+                label="title"
+                outlined
+                >
```

Comment:
nit; indentation, the correct indentation will improve the readability.
Please run `npm run lintfix` before committing